AssemblerLib/LocalDataInitializer.h

```diff
@@ -119,35 +119,17 @@ namespace AssemblerLib
 /// For example for MeshLib::Line a local assembler data with template argument
 /// NumLib::ShapeLine2 is created.
 template <
-    template <typename, typename> class LocalAssemblerDataInterface_,
-    template <typename, typename, typename, typename, unsigned> class LocalAssemblerData_,
-    typename GlobalMatrix_,
-    typename GlobalVector_,
+    class LocalAssemblerInterface,
+    template <typename, typename, typename, typename, unsigned> class LocalAssemblerData,
+    typename GlobalMatrix,
+    typename GlobalVector,
     unsigned GlobalDim,
     typename... ConstructorArgs>
 class LocalDataInitializer
 {
-    template <typename ShapeFunction_>
-    using IntegrationMethod = typename NumLib::GaussIntegrationPolicy<
-                typename ShapeFunction_::MeshElement>::IntegrationMethod;
-
-    template <typename ShapeFunction_>
-    using LAData = LocalAssemblerData_<
-            ShapeFunction_,
-            IntegrationMethod<ShapeFunction_>,
-            GlobalMatrix_, GlobalVector_, GlobalDim>;
-
-    using LADataIntfPtr
-        = std::unique_ptr<LocalAssemblerDataInterface_<GlobalMatrix_, GlobalVector_>>;
-
-    using LADataBuilder = std::function<LADataIntfPtr(
-            MeshLib::Element const& e,
-            std::size_t const local_matrix_size,
-            unsigned const integration_order,
-            ConstructorArgs&&...
-        )>;
-
 public:
+    using LADataIntfPtr = std::unique_ptr<LocalAssemblerInterface>;
```

Comment:
This line is only move and not introduced in this PR. And it is used in four places, or so.